lib/node_options.js

```diff
@@ -23,11 +23,18 @@ class NodeOptions {
   /**
    * Create a new instance with default property values.
    * @constructor
+   * @param {boolean} [startParameterServices=true]
+   * @param {array} [parameterOverrides=[]]
+   * @param {boolean} [automaticallyDeclareParametersFromOverrides=false]
    */
-  constructor() {
-    this._startParameterServices = true;
-    this._parameterOverrides = [];
-    this._automaticallyDeclareParametersFromOverrides = false;
+  constructor(
+    startParameterServices = true,
+    parameterOverrides = [],
+    automaticallyDeclareParametersFromOverrides = false
```

Comment:
@wayneparrott I want to leverage the default parameters to init the members here, PTAL

Reply:
I'm cool with this change. LGTM